lib/cloud/aws/tags.go

```diff
@@ -42,3 +56,69 @@ func IsTagValueTrue(value string) bool {
 		return true
 	}
 }
+
+// SettableTag is a generic interface that represents an AWS resource tag with
+// SetKey and SetValue functions.
+type SettableTag[T any] interface {
+	SetKey(key string) *T
+	SetValue(Value string) *T
+	*T
+}
+
+// LabelsToTags converts a label map to a list of AWS resource tags.
+func LabelsToTags[T any, PT SettableTag[T]](labels map[string]string) (tags []*T) {
```

Comment:
Right now this function is used only in test Can we mark this function as tags_helpers.go  ?

Reply:
I really want `LabelsToTags` and `TagsToLabels` stay together so I moved both to the helper file.